README.adoc

```diff
@@ -574,6 +574,30 @@ class SomethingTest < Minitest::Test
 end
 ----
 
+=== Skipping [[skipping]]
+
+Prefer `skip` over `return` for skipping test cases.
+
+[source,ruby]
+----
+
+# bad
+
+def test_something
+  return unless condition?
+
+  # do testing
+end
+
+# good
+
+def test_something
+  skip("Condition not satisfied") unless condition?
```

Comment:
It's polite to have comment, but it can be omitted in the example according to https://docs.rubocop.org/rubocop-minitest/cops_minitest.html#minitestskipwithoutreason.

```suggestion
  skip if condition?
```